packages/icon-button/src/mwc-icon-button.scss

```diff
@@ -91,15 +89,7 @@ limitations under the License.
 }
 
 // hack for top-app-bar ripple coloring of icon-buttons
-.mdc-ripple-upgraded {
-  &:focus::before, &:focus::after {
-    @include theme.prop(background-color, (
-      varname: --mdc-theme-on-primary,
-      fallback: currentColor,
-    ));
-    @include theme.prop(opacity, (
-      varname: --mdc-icon-button-ripple-opacity,
-      fallback: 0.12,
-    ));
-  }
+mwc-ripple {
+  --mdc-ripple-color: var(--mdc-theme-on-primary, currentColor);
```

Comment:
@asyncLiz should this use the custom-props pattern or not since we're probably going to refactor this file anyway?

Reply:
Would that be:
```
    @include theme.property(--mdc-ripple-color, (
      varname: --mdc-theme-on-primary,
      fallback: currentColor,
    ));
```